spanner/spanner_snippets/snippet_test.go

```diff
@@ -239,6 +238,9 @@ func TestSample(t *testing.T) {
 	out = runCommand(t, "querywithstring", dbName)
 	assertContains(t, out, "42 Venue 42")
 
+	// Wait 5 seconds to avoid a time drift issue for the
+	// next query.
```

Comment:
Would you mind linking to the issue in this comment? Would be nice to have the extra context.